### ISSUE TITLE
AirCurve 11 not detected ("Unknown")

### ISSUE THREAD

**Issue by Webster83** [VibeCoder75321/Multi-Night-Glasgow-Index-Analyzer#13]
The parseMachineIdentifcation() definition looks for jsonData.ProductName, .SerialNumber and .SoftwareID. On the AC11, these keys are not located at root. Instead, they are located under .FlowGenerator.IdentificationProfiles.Product (for name and serial number). 

The following code change should address this case:

```javascript
                    machineInfo = {
                        type: jsonData.ProductName || jsonData.FlowGenerator.IdentificationProfiles.Product.ProductName || 'Unknown',
                        serial: jsonData.SerialNumber || jsonData.FlowGenerator.IdentificationProfiles.Product.SerialNumber|| 'Unknown',
                        softwareId: jsonData.SoftwareId || jsonData.FlowGenerator.IdentificationProfiles.Software.ApplicationIdentifier || 'Unknown'
``` 

I validated `console.log("ParsedJSON: ",jsonData)` was reading the Identification.json file so it was simply a matter of capturing the case where these keys do exist, just not in the 'A' case so rather than A||"Unknown", we need an A||B||'Unknown'
